cms/release_calendar/models.py

```diff
@@ -143,6 +143,7 @@ class ReleaseCalendarPage(BasePage):  # type: ignore[django-manager-missing]
     search_fields: ClassVar[list[index.SearchField | index.AutocompleteField | index.FilterField]] = [
```

Comment:
Minor point, but all index field types inherit from `wagtail.search.index.BaseField`, which could simplify type hints.

Reply:
it is a good point, so updated everywhere bd13d71